setup.py

```diff
@@ -9,7 +9,7 @@
 """
 
 import os
-from setuptools import setup, find_packages
+from setuptools import setup, find_packages  # type: ignore
```

Comment:
what was the issue here ? Mind adding a comment that explains ?

Reply:
Mypy claims setuptools is missing type annotations. Maybe a 3.6 error?